### PR TITLE
Adds Antag Tokens!

### PR DESCRIPTION
**Pull request by Incoming5643**
They're a novelty coin with a syndie helmet (valid) on one side and a salad (salad) on the other.
Found as a prize in arcade machines, Sec can also get one from their vendor as a premium item.

d10_icon
coin

### CODE REVIEW

**Comment by optimumtact** [comment]
:+1:

**Comment by Supermichael777** [comment]
can they be used in vending machines? both as a balance issue and cool factor.

**Comment by Incoming5643** [comment]
They're coins, so yeah you can use them in vending machines if you really want to. That's why getting one from the sec machine takes a coin.

**Comment by PKPenguin321** [comment]
Add it to the traitor uplink to make the Two-Face gimmick that much more intimidating

File: code/modules/mining/ores_coins.dm [hunk 1]
```diff
@@ -280,6 +280,16 @@
 	sideslist = list("heads")
 	value = 20
 
+/obj/item/weapon/coin/antagtoken
+	name = "antag token"
+	icon_state = "coin_valid_valid"
+	cmineral = "valid"
+	desc = "A novelty coin that helps the heart know what hard evidence cannot prove."
+	sideslist = list("valid", "salid")
```
Comment (Dorsisdwarf) [inline code/modules/mining/ores_coins.dm]: What is even up with "salid" anywhere in this?
Should be valid salad, shouldn't it?

Comment (Incoming5643) [inline code/modules/mining/ores_coins.dm]: It's obvious I don't eat enough salad

**Pull request merged**